OneSignalSDK/onesignal/src/main/java/com/onesignal/NotificationRestorer.java

```diff
@@ -191,11 +191,11 @@ static void showNotificationsFromCursor(Context context, Cursor cursor, int dela
       if (!cursor.moveToFirst())
          return;
 
-      boolean useExtender = (OSNotificationIntentService.getIntent(context) != null);
+      boolean useExtender = false; //OSNotificationIntentService.getIntent(context) != null;
```

Comment:
is this ok? useExtender is never updated, so it will never enter on the if condition

Reply:
This is something else not related to Android X migration